src/panels/lovelace/editor/config-elements/hui-history-graph-card-editor.ts

```diff
@@ -44,7 +53,7 @@ export class HuiHistoryGraphCardEditor extends LitElement
   @internalProperty() private _configEntities?: EntityConfig[];
 
   public setConfig(config: HistoryGraphCardConfig): void {
-    config = cardConfigStruct(config);
+    assert(config, cardConfigStruct);
```

Comment:
:( wish Superstruct didn't change this, the old one was fine.

Reply:
The replacement for this is `const user = coerce(data, User)` I guess, but we don't need/use that in most structs 🤷‍♂️